dags/imagegen_devx/configs/project_bite_config.py

```diff
@@ -28,9 +28,9 @@
 def set_up_axlearn() -> Tuple[str]:
   return (
       common.UPGRADE_PIP,
-      *common.set_up_nightly_jax(),
-      "python -m pip install axlearn",
       "git clone https://github.com/apple/axlearn.git",
+      "python -m pip install axlearn",
```

Comment:
Should this be
```suggestion
      "python -m pip install ./axlearn",
```
Otherwise it will install the package in pypi?

Reply:
SG! This `./axlearn` points to dependencies [here](https://github.com/apple/axlearn/blob/main/pyproject.toml#L14-L38) similar to `python -m pip install .` right?

Reply:
Updated.

Reply:
cc @markblee @jiya-zhang

Reply:
Thanks!

> This ./axlearn points to dependencies [here](https://github.com/apple/axlearn/blob/main/pyproject.toml#L14-L38) similar to python -m pip install . right?

That's right.

> They return the same error now (with nightly) for PJRT version conflict.

Where can I see this error?

Reply:
Hi @markblee, we noticed our internal JAX tests also meet this issue recently.

I have some logs to the PR description. In the future, @jiya-zhang and I were discussing to grant you proper permissions to full logs from GCS bucket so that you can see logs directly from test runs from our automation platform. Thanks!